gum/gumeventsink.c

```diff
@@ -60,3 +78,41 @@ gum_event_sink_stop (GumEventSink * self)
   if (iface->stop != NULL)
     iface->stop (self);
 }
+
+static void
+gum_default_event_sink_class_init (GumDefaultEventSinkClass * klass)
+{
+}
+
+static void
+gum_default_event_sink_iface_init (gpointer g_iface,
+                                   gpointer iface_data)
+{
+  GumEventSinkInterface * iface = g_iface;
+
+  iface->query_mask = gum_default_event_sink_query_mask;
+  iface->process = gum_default_event_sink_process;
+}
+
+static void
+gum_default_event_sink_init (GumDefaultEventSink * self)
+{
+}
+
+static GumEventType
+gum_default_event_sink_query_mask (GumEventSink * sink)
+{
+  return GUM_NOTHING;
+}
+
+static void
+gum_default_event_sink_process (GumEventSink * sink,
+                                const GumEvent * ev)
+{
+}
+
+GumEventSink *
+gum_event_sink_make_default (void)
```

Comment:
Let's move this right before `gum_default_event_sink_class_init()` as this is higher level so it belongs higher up.